src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -40,7 +40,25 @@ export const ResultsList = ({
   query,
   ...props
 }) => {
+  const [resultsData, setResultsData] = useState(null);
   const searchResultTitle = useRef();
+  const markers = MARKER_LETTERS.values();
+  const currentPage = pagination ? pagination.currentPage : 1;
+
+  useEffect(
```

Comment:
It was necessary to move the setting results data to a `useEffect` because it was previously being called multiple times causing unnecessary re-renders on the results and multiple GA events to fire.
 
Screenshots of prod behavior
<img width="600" alt="Screenshot 2024-08-08 at 12 01 59 PM" src="https://github.com/user-attachments/assets/cba8c00a-bcaf-46d9-9712-8bc4f5f20cad">


<img width="600" alt="Screenshot 2024-08-08 at 11 59 32 AM" src="https://github.com/user-attachments/assets/9f049801-f593-4274-99ff-6b8fcc90480d">